### ISSUE TITLE
Scene.RayCount crashes on 32bit systems with cgo

### ISSUE THREAD

**Issue by therecipe** [fogleman/pt#16]
Hey

First thank you for this lib and also for your other lib `ln` :)
I had great fun playing around with the examples and even build an example on top of your examples for my lib here: https://github.com/therecipe/qt/tree/master/internal/examples/widgets/renderer

But it seems like the examples crash if you compile them for windows/386 in combination with cgo.
I tracked the problem down and wrote some about it here: https://github.com/therecipe/qt/issues/125

Basically to fix this you would just have to re-order this struct https://github.com/fogleman/pt/blob/master/pt/scene.go#L5-L13
and simply move `rays` to the top.

I hoped that you would fix this, so that I don't have to send a PR for such a small change.
